packages/metal-clay-button/src/MetalClayButton.js

```diff
@@ -61,11 +61,10 @@ MetalClayButton.STATE = {
 	 * @instance
 	 * @memberof MetalClayButton
 	 * @type {?string}
-	 * @default 'default'
+	 * @default undefined
 	 */
 	style: {
 		validator: validators.string,
-		value: 'default'
```

Comment:
I thought more about it and just update the JSDoc to `* @type {?string|undefined}` works. When we define an attribute value here that means that it is its default value, that is not the case here, there is no default value anymore, then makes sense return `undefined` here.

Does it make sense, @brunobasto?